pkg/types/chains/evm/evm.go

```diff
@@ -108,3 +114,51 @@ type Head struct {
 type TransactionFee struct {
 	TransactionFee *big.Int // Cost of transaction in wei
 }
+
+type SignedReport struct {
+	RawReport     []byte
+	ReportContext []byte
+	Signatures    [][]byte
+	ID            []byte
+}
+
+// TransactionStatus is the result of the transaction sent to the chain
+type TransactionStatus int
+
+const (
+	//Transaction was sent successfully to the chain and successfully executed
+	TxSuccess TransactionStatus = iota
```

Comment:
It might be risky to have Success be the default zero value. Is there a better option? What is the plan for Pending? That would be an improvement.